unity-renderer/Assets/DCLPlugins/EmoteAnimations/EmoteAnimationsTracker.cs

```diff
@@ -58,7 +58,8 @@ private void InitializeEmbeddedEmotes()
                     //Unity's Animation uses the name to play the clips.
                     embeddedEmote.maleAnimation.name = embeddedEmote.id;
                     dataStore.emotesOnUse.SetRefCount((MALE,  embeddedEmote.id), 5000);
-                    dataStore.animations.Add((MALE, embeddedEmote.id), embeddedEmote.maleAnimation);
+                    var clipData = new EmoteClipData(embeddedEmote.maleAnimation);
+                    dataStore.animations.Add((MALE, embeddedEmote.id), clipData);
```

Comment:
minor comment, because clipData is not used anywhere else could you change the addition to 
```
dataStore.animations.Add((MALE, embeddedEmote.id), new EmoteClipData(embeddedEmote.maleAnimation));
```
so we avoid additional variables, as you did below in the tests?